Qpf/Qpf/Multivariate/ofPolynomial.lean

```diff
@@ -11,27 +11,39 @@ import Qpf.Macro.Tactic.FinDestr
 -/
 
 namespace MvQPF
+  def ofIsomorphismMvFunctor {F : TypeFun n} 
+                    (F' : TypeFun n)
+                    [q : MvFunctor F']
+                    (box    : ∀{α}, F α → F' α) 
+                    (unbox  : ∀{α}, F' α → F α) 
+                  : MvFunctor F
+    where
+      map f a     := unbox <| q.map f <| box a
+
   /-- If `F` is isomorphic to a QPF `F'`, then `F` is also a QPF -/
   def ofIsomorphism {F : TypeFun n} 
                     (F' : TypeFun n)
+                    [func' : MvFunctor F']
                     [q : MvQPF F']
                     (box    : ∀{α}, F α → F' α) 
                     (unbox  : ∀{α}, F' α → F α) 
                     (box_unbox_id : ∀{α} (x : F' α), box (unbox x) = x)
                     (unbox_box_id : ∀{α} (x : F α), unbox (box x) = x 
                                   := by intros; rfl
                                 )
+                    (func : MvFunctor F)
+                    (func_map : ∀ (α β : TypeVec n) (f : TypeVec.Arrow α β) (a : F α), func.map f a = (unbox <| func'.map f <| box a))
```

Comment:
Why do we need these extra arguments?

Reply:
To be able to return `MvQPF F`, we need to make sure that we have `MvFunctor F`. To make `abs_map` work, we want to ensure that this instance is particularly the one that we get from `MvFunctor.ofIsomorphism` above. I decided to express this assertion by passing a prop for the necessary equality. Maybe there is a better way of achieving this :)

Reply:
Ah, right. This used to be bundled, but now that we're using normal Mathlib again, we need the instance to be able to write `MvQPF F`. We could avoid this, by passing the instance explicitly, as in `@MvQPF _ F (MvFunctor.ofIsomorphism ..)`, but probably it's nicer to allow the possibility of a different instance as well.

However, I would suggest we make have the instance be inferred, and use an auto-tactic for the proof. This way, in the common case we don't actually have to supply anything.
```suggestion
                    [functor : MvFunctor F]
                    (map_eq : ∀ (α β : TypeVec n) (f : TypeVec.Arrow α β) (a : F α), functor.map f a = (unbox <| func'.map f <| box a) := by intros; rfl)
```

Reply:
I'll give it a shot :) 
I'll also give proper names in places where I used `func`. I did this all pretty quick and dirty for now and did not adjust afterwards. Sorry for that :D

Reply:
Alright, I did the adjustment, also in `Qpf/Qpf/Multivariate/Basic.lean`. Everything seems to work with the implicit argument now :)